pkg/controllers/management/auth/indexes_test.go

```diff
@@ -0,0 +1,271 @@
+package auth
+
+import (
+	"reflect"
+	"testing"
+
+	v1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func Test_getRBOwnerKey(t *testing.T) {
+	type args struct {
+		rb *v1.RoleBinding
+	}
+	tests := []struct {
+		name string
+		args args
+		want []string
+	}{
+		{
+			name: "get_two_ownerreferences",
```

Comment:
Small tip...you don't need to make these into snake-case, you can use spaces between words, the Go test runner will do it.

https://cs.opensource.google/go/go/+/refs/tags/go1.22.5:src/testing/match.go;l=282

Reply:
thanks! I'll fix it :)